cliquet/__init__.py

```diff
@@ -16,6 +16,7 @@
 from cliquet.initialization import (  # NOQA
     initialize, initialize_cliquet, install_middlewares)
 
+__all__ = ['utils', 'errors', 'resource']
```

Comment:
If the aim is to do ``from cliquet import ResourceSchema`` then we should have a list of all the think we want to expose in ``cliquet``

Something like: ``__all__ = ['ResourceSchema', 'PermissionsSchema', 'register', 'Service', 'DEFAULT_SETTINGS', ...]``